assets-and-images.md

```diff
@@ -206,3 +206,48 @@ level classes, like
 or
 [`ImageCache`](https://docs.flutter.io/flutter/services/ImageCache-class.html),
 you'll also notice parameters related to scale.)
+
+## Platform assets
+
+There will also be occasions to work with assets in the platform projects directly. Below are 2 common cases where assets are used before the Flutter framework is loaded and running.
+
+### Updating the app icon
+Updating your Flutter application's launch icon works the same way as updating launch icons in native Android or iOS applications.
+
+![Launch icon](/images/assets-and-images/icon.png)
+
+#### Android
+
+In your Flutter project's root directory, navigate to `.../android/app/src/main/res`. The various bitmap resource folders such as `mipmap-hdpi` already contain placeholder images named `ic_launcher.png`. Simply replace them with your desired assets respecting the recommended icon size per screen density as indicated by the [Android Developer Guide](https://developer.android.com/guide/practices/ui_guidelines/icon_design_launcher.html#size).
+
+![Android icon location](/images/assets-and-images/android-icon-path.png)
+
+Note: if you rename the .png files, you must also match the new name in your `AndroidManifest.xml`'s `<application>` tag's `android:icon` attribute.
```

Comment:
I actually wanted to achieve the effects found in some of the other pages, i.e. users don't have to read this if they're just following along. Added that <aside> box.